README.md

```diff
@@ -75,6 +76,8 @@ var myOverlay = new Overlay('myOverlay', {
 * `zindex`: String. Value of the CSS z-index property of the overlay. _Default set via CSS_: '10'
 * `preventclosing`: Boolean. Prevents closure of overlay via standard x button or escape key. For use when you are directing the user to somewhere else. Only valid with modal set to true.
 * `customclose`: Boolean. If you do not use the header, but want to provide a close button in your html / src (with a class of `o-overlay__close`), setting customclose to true will attach o-overlay's close handler function to that button.
+* `parentNode`: String. Should be a query selector for a DOM element. If set, the overlay will be appended as a child of this rather than the document body or target. If multiple nodes are matched, it will use the first. If nothing matches this selector, the `body` will be used.
```

Comment:
Would this option ever be used with the `arrow` option? If it would, then if the query selector does not match anything than `body` will not be the element used, it would be `oLayers.getLayerContext(this.opts.arrow.target)`.